### DISCUSSION TITLE
Add an option to delete a worker node in the TEAM creation view

### DISCUSSION THREAD

**Discussion by acherla** [StreetLamb/tribe#135] [Ideas]
When creating a team and creating a worker within that team, provide the ability to delete that worker/leader node as a button on the UI

Benefit: Gives users the ability to delete workers that they may have created accidently or no longer need as a part of a sequential or hierarchical workflow.  

Implementation:
1. User clicks delete button on node
2. Receive a modal dialogue that verifies if the user wants to perform this delete action? (Delete/Cancel)
3. If its a leader node, delete all worker nodes that roll up to the root node and the root node itself, 
4. if its a worker node delete the individual node

**Comment by johnkm516** [comment]
Came to ask the same thing. Spent a while looking for the Delete node only to realize it doesn't exist.

**Reply by StreetLamb** [reply]
Hey @johnkm516, currently you can delete a node by selecting it and pressing the delete or backspace key on your keyboard. But yes, we should add a delete button so it is clearer.

**Reply by johnkm516** [reply]
@StreetLamb Ah the hotkey was backspace! I tried pressing the delete button and it wouldn't work. Since the delete node wasn't in the UI I assumed the function didn't exist. Thank you for the help!
